### ISSUE TITLE
Example needs renaming to match lib name

### ISSUE THREAD

**Issue by kattni** [adafruit/Adafruit_CircuitPython_MSA301#1]
Example `tap_example.py` should be named `msa301_tap_example.py` to match CircuitPython library standards and clarify which library it is associated with in the example bundle.

The lines including this file name in the `docs/examples.rst` file must also be updated to reflect the name change, e.g.:
```
.. literalinclude:: ../examples/current_file_name.py
    :caption: examples/current_file_name.py
    :linenos:
```
would be updated to:
```
.. literalinclude:: ../examples/current_file_name_changed.py
    :caption: examples/current_file_name_changed.py
    :linenos:
```

**Comment by geekguy-wy** [comment]
If there is no mention of *tap_example.py in docs/examples.rst, it would need to be added, right?

**Comment by kattni** [comment]
@geekguy-wy That is correct! If the example is not listed, it would need to be added.

**Comment by kattni** [comment]
This has been resolved by #4 and #5
